company_developers8.txt

```diff
@@ -9556,7 +9556,6 @@ NEC Corporation:
 	oomichi: ken-oomichi!wx.jp.nec.com, oomichi!users.noreply.github.com until 2022-06-01
 	parauliya: parauliya!users.noreply.github.com until 2017-01-01
 	poothia: poothia!users.noreply.github.com, poothia.shubhendu!gmail.com, shubhendu.poothia!india.nec.com until 2019-08-01
-	pranav-pandey0804: pranav-pandey0804!users.noreply.github.com, pranav.pandey!india.nec.com
```

Comment:
you should not remove the entry instead add the end date in all the file

pranav-pandey0804: pranav-pandey0804!users.noreply.github.com, pranav.pandey!india.nec.com
	NEC Corporation until 2024-11-08
	Independent from 2024-11-09

Example- https://github.com/cncf/gitdm/commit/e87b8162c9cbe8370611eacf4e99c117b39166b6

Reply:
I will handle this manually.